app/lib/results.rb

```diff
@@ -4,8 +4,10 @@ def initialize(left, right)
     @result_count = right['results'].count > left['results'].count ? right['results'].count : left['results'].count
     @left = (0..result_count - 1).map { |i| Result.new(left['results'][i]) }
     @right = (0..result_count - 1).map { |i| Result.new(right['results'][i]) }
-    @left_total = left['total']
-    @right_total = right['total']
+
+    @left_total = left['total'] || left['result_count']
+    @right_total = right['total'] || right['result_count']
```

Comment:
Would like to understand this more as we should have the same output from all environments. Let's sit down and review